module-services/service-gui/service-gui/ServiceGUI.hpp

```diff
@@ -67,24 +64,18 @@ namespace service::gui
         bool isAnyFrameBeingRenderedOrDisplayed() const noexcept;
         void trySendNextFrame();
 
-        void setState(State state) noexcept;
-        bool isInState(State state) const noexcept;
-
         sys::MessagePointer handleDrawMessage(sys::Message *message);
         sys::MessagePointer handleGUIRenderingFinished(sys::Message *message);
-        sys::MessagePointer handleEinkInitialized(sys::Message *message);
         sys::MessagePointer handleImageDisplayedNotification(sys::Message *message);
         sys::MessagePointer handleChangeColorScheme(sys::Message *message);
 
+        ::gui::Size displaySize;
```

Comment:
why is the displaySize added to ServiceGui in this PR?

Reply:
It was added because before display size was fetched from `EinkInitialized` message. This message was removed altogether as it is no longer needed, thanks to the properly defined dependencies. From now, such information is injected through service's constructor.